sql/core/src/main/scala/org/apache/spark/sql/execution/python/BatchEvalPythonExec.scala

```diff
@@ -69,7 +67,7 @@ case class BatchEvalPythonExec(udfs: Seq[PythonUDF], output: Seq[Attribute], chi
 
     // Output iterator for results from Python.
     val outputIterator = new PythonUDFRunner(
-        funcs, bufferSize, reuseWorker, PythonEvalType.SQL_BATCHED_UDF, argOffsets)
+        funcs, PythonEvalType.SQL_BATCHED_UDF, argOffsets, sparkContext.conf)
```

Comment:
I _think_ this code is executed on the workers, which means you can't access the SparkContext here which could explain the local test failures.

Reply:
Good catch, I'll fix it.